[BUG] RAPIDS-accelerated-UDFs native test fails with undefined symbol cudf::copy_bitmask (resource_ref ABI mismatch)

**Describe the bug**
Build: examples-udf-examples-native/583

The native UDF integration test `test_hive_simple_udf_native` aborts the JVM when loading `udfexamplesjni*.so`. The loader reports an undefined symbol for `cudf::copy_bitmask(column_view const&, rmm::cuda_stream_view, cuda::mr::resource_ref<device_accessible>)`, indicating an ABI mismatch between the native UDF shared library and the cudf library bundled in the rapids-4-spark plugin jar (both reported as 26.06.0-SNAPSHOT).

Error logs:
```
src/main/python/rapids_udf_test.py::test_hive_simple_udf_native
/usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java: symbol lookup error: /tmp/udfexamplesjni8858388526270039482.so: undefined symbol: _ZN4cudf12copy_bitmaskERKNS_11column_viewEN3rmm16cuda_stream_viewEN4cuda2mr3__420__version_bump_ver4_12resource_refIJNS7_17device_accessibleEEEE
script returned exit code 127

RapidsPluginUtils: RAPIDS Accelerator build: version -> 26.06.0-SNAPSHOT, cudf_version -> 26.06.0-SNAPSHOT
RapidsPluginUtils: cudf build: version -> 26.06.0-SNAPSHOT
```

**Environment details**
- Spark 3.3.0 (shim spark330)
- rapids-4-spark 26.06.0-SNAPSHOT, cudf 26.06.0-SNAPSHOT
- JDK 8, Python 3.10
- CUDA 12 dev image, native UDF build (`-Pudf-native-examples`)

--- a/examples/UDF-Examples/RAPIDS-accelerated-UDFs/Dockerfile
+++ b/examples/UDF-Examples/RAPIDS-accelerated-UDFs/Dockerfile
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2021-2025, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2021-2026, NVIDIA CORPORATION. All rights reserved.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
@@ -20,8 +20,8 @@ ARG LINUX_VERSION=rockylinux8
 
 FROM nvidia/cuda:${CUDA_VERSION}-devel-${LINUX_VERSION}
 
-ARG TOOLSET_VERSION=14
-ENV TOOLSET_VERSION=14
+ARG TOOLSET_VERSION=13
+ENV TOOLSET_VERSION=13
 ARG PARALLEL_LEVEL=10
 ENV PARALLEL_LEVEL=10
 

--- a/examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt
+++ b/examples/UDF-Examples/RAPIDS-accelerated-UDFs/src/main/cpp/CMakeLists.txt
@@ -18,7 +18,7 @@
 cmake_minimum_required(VERSION 3.30.4 FATAL_ERROR)
 
 # set to the rapids-cmake-branch
-set(rapids-cmake-branch "release/26.02")
+set(rapids-cmake-branch "main")
 
 file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/${rapids-cmake-branch}/RAPIDS.cmake
      ${CMAKE_BINARY_DIR}/RAPIDS.cmake)

